build.gradle.kts

```diff
@@ -2,6 +2,10 @@ plugins {
     id("com.gtnewhorizons.gtnhconvention")
 }
 
+minecraft {
+    extraRunJvmArguments.add("-Dangelica.enableTestBlocks=true")
```

Comment:
local changeset?

Reply:
I left them in because I assume you'd want to have the test block enabled in dev.